tests/e2e/Skip_Initial_Tests/tests/CanaryTest.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace ProvideExistingCoverage\Test;
+
+use PHPUnit\Framework\TestCase;
+use ProvideExistingCoverage\SourceClass;
+
+/** @coversNothing */
+class CanaryTest extends TestCase
+{
+    public function test_it_makes_a_file(): void
+    {
+        $this->assertSame(7, file_put_contents(__DIR__ . '/../has_run', 'phpunit'));
+    }
```

Comment:
I'm not even sure how this test was working before, but now we can be absolutely certain that this file is only going to be created during initial run: this tests covers nothing, therefore no mutation can trigger it.